library/src/main/kotlin/com/thefuntasty/donut/DonutDataset.kt

```diff
@@ -0,0 +1,11 @@
+package com.thefuntasty.donut
+
+/**
+ * Data class representing dataset containing [Float] entries, color and name.
+ * Entries with the same [name] are shown on same progress line.
+ */
+data class DonutDataset(
+    val name: String,
+    val color: Int,
+    val entries: List<Float>
```

Comment:
Why do you hold entries as a list of floats instead of single float? It makes sense to me, when there is some `Entry` class and wraps float value with some additional value (timestamp, ...), but thats not this case.

Reply:
There was a `DonutProgressEntry` class that also held color and other stuff, but it basically became a Float wrapper after refactor to datasets, so I just removed it and used Float list instead. The view was originally designed to work with entry list and I actually thought about flattening this list to a single value, but I don't remember why I didn't. Maybe because dataset  naming suggests holding multiple data entries?
Anyway, I can flatten the list to a single entry and the view will become what was before refactor, or re-introduce the `ProgressEntry` class to hold some more interesting data besides it's value?

Reply:
@skywall I implemented it like you suggested - list of floats would not make any sense - user will be required to provide sum of all entries in dataset to the view